src/module.c

```diff
@@ -13608,19 +13644,46 @@ int moduleDefragValue(robj *key, robj *value, int dbid) {
     return 1;
 }
 
+#define modulesForEach(...) do { \
```

Comment:
i would prefer to have a `dictForEach` macro. 🙃
like dictForEach(dict, struct RedisModule, member, {});

Reply:
Thanks, done.